### PR TITLE
Add more delete rules and resource types for janitor

### PR DESCRIPTION
**Pull request by krzyzacy**
So forwarding-rules and backend-services can be either `--global` or `--region=REGION`.

And http-proxy and url-map need to be killed before shut down backend-services

```
NAME
    gcloud compute backend-services delete - delete backend services

SYNOPSIS
    gcloud compute backend-services delete NAME [NAME ...]
        [--global | --region=REGION] [GLOBAL-FLAG ...]
```
Also fixed a bunch of lint errors add it to pylint list.

#1573

### CODE REVIEW

**Comment by fejta** [comment]
@spxtr will you assign this back to me once bazel works again?

**Comment by spxtr** [comment]
Sure

**Comment by spxtr** [comment]
@k8s-bot bazel test this

**Comment by krzyzacy** [comment]
@k8s-bot bazel test this

**Comment by spxtr** [comment]
@k8s-bot bazel test this

**Comment by krzyzacy** [comment]
now you crossed that magic 15 :-)

**Comment by krzyzacy** [comment]
@k8s-bot bazel test this

File: verify/verify-pylint.sh [hunk 1]
```diff
@@ -20,5 +20,6 @@ set -o pipefail
 # TODO(fejta): all python files
 pylint scenarios/*.py
 pylint jenkins/bootstrap.py
+pylint jenkins/janitor.py
```
Comment (fejta) [inline verify/verify-pylint.sh]: Yay!

File: jenkins/janitor.py [hunk 2]
```diff
@@ -50,34 +55,44 @@ def collect(project, age, resource, filt):
...
             '--project=%s' % project])):
 
-        if not item['name'] or not item['creationTimestamp']:
-            print '[Warning] - Skip item without name or timestamp'
-            print '%s' % item
-            continue
-        if resource.condition and not item[resource.condition]:
-            print '[Warning] - condition specified but not found'
-            continue
+        print '%s' % item
```
Comment (fejta) [inline jenkins/janitor.py]: nit: consider refactoring this to use logging instead of print (fine to punt to a future pr)

Comment (krzyzacy) [inline jenkins/janitor.py]: I'll make a separate PR then

**Pull request merged**